Add logging to inliner.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29921 Add a pass that reconstructs scopes for nodes from their inline callstacks.
* #29911 Show full call stack in TorchScript exception even when calls were inlined.
* **#27922 Add logging to inliner.**
* #27921 Add InlinedCallStack class.
* #27920 Make inlineCallTo to take Function instead of Graph as the callee argument.
* #27919 Add a variant of insertGraph that fills values map.

gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

Commit:
Add logging to inliner.

Commit:
Update on "Add logging to inliner."

Add logging to inliner.

gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
rebase on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

kostmo commented:
## CircleCI build failures summary
As of commit 81884aa:
* 0/8 recognized as flaky
* 8/8 broken upstream. You may want to rebase on the [latest viable branch](https://dr.pytorch.org/master-viable-commits.html).
* 0/8 failures introduced in this PR



Here are the [reasons each build failed](https://dr.pytorch.org/commit-details.html?sha1=81884aa143577cdeb32964e8434d434279472636).

---
This comment was automatically generated by [Dr. CI](https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment).
Follow [this link to opt-out](https://dr.pytorch.org/admin/comments-opt-out.html) of these comments for your Pull Requests.


Please report bugs/suggestions on the [GitHub issue tracker](https://github.com/kostmo/circleci-failure-tracker/issues).

Krovatkin reviewed (approved):
:shipit:

Commit:
rebase on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head

Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
Update on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]

Commit:
rebase on "Add logging to inliner."


gh-metadata: pytorch pytorch 27922 gh/ZolotukhinM/140/head


Differential Revision: [D17914135](https://our.internmc.facebook.com/intern/diff/D17914135)

[ghstack-poisoned]
